### ISSUE TITLE
Error building on Julia 1.3.0, Win10

### ISSUE THREAD

**Issue by islent** [JuliaGizmos/Blink.jl#223]
Here's the error:
```julia
(v1.3) pkg> build Blink
  Building MbedTLS → `C:\Users\islent\.julia\packages\MbedTLS\a1JFn\deps\build.log`
  Building WebIO ──→ `C:\Users\islent\.julia\packages\WebIO\2mZPb\deps\build.log`
  Building Blink ──→ `C:\Users\islent\.julia\packages\Blink\AO8uN\deps\build.log`
┌ Error: Error building `Blink`: 
│ ERROR: LoadError: ArgumentError: reducing over an empty collection is not allowed
│ Stacktrace:
│  [1] _empty_reduce_error() at .\reduce.jl:212
│  [2] reduce_empty(::Function, ::Type) at .\reduce.jl:222
│  [3] mapreduce_empty(::typeof(identity), ::Function, ::Type) at .\reduce.jl:247
│  [4] _mapreduce(::typeof(identity), ::typeof(max), ::IndexLinear, ::Array{VersionNumber,1}) at .\reduce.jl:301
│  [5] _mapreduce_dim at .\reducedim.jl:312 [inlined]
│  [6] #mapreduce#584 at .\reducedim.jl:307 [inlined]
│  [7] mapreduce at .\reducedim.jl:307 [inlined]
│  [8] _maximum at .\reducedim.jl:657 [inlined]
│  [9] _maximum at .\reducedim.jl:656 [inlined]
│  [10] #maximum#591 at .\reducedim.jl:652 [inlined]
│  [11] maximum at .\reducedim.jl:652 [inlined]
│  [12] (::Main.BuildBlink.var"#1#4")() at C:\Users\islent\.julia\packages\Blink\AO8uN\src\AtomShell\install.jl:66
│  [13] cd(::Main.BuildBlink.var"#1#4", ::String) at .\file.jl:93
│  [14] install() at C:\Users\islent\.julia\packages\Blink\AO8uN\src\AtomShell\install.jl:27
│  [15] top-level scope at C:\Users\islent\.julia\packages\Blink\AO8uN\deps\build.jl:14
│  [16] include at .\boot.jl:328 [inlined]
│  [17] include_relative(::Module, ::String) at .\loading.jl:1105
│  [18] include(::Module, ::String) at .\Base.jl:31
│  [19] include(::String) at .\client.jl:424
│  [20] top-level scope at none:5
│ in expression starting at C:\Users\islent\.julia\packages\Blink\AO8uN\deps\build.jl:13
└ @ Pkg.Operations D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\backwards_compatible_isolation.jl:649
```
Files in `deps/` :
![image](https://user-images.githubusercontent.com/31057826/70025184-36d06380-15d7-11ea-80a6-7c4bdb6c74f0.png)

**Comment by BoundaryValueProblems** [comment]
I'm facing exactly the same problem on my Windows 10 machine.

**Comment by travigd** [comment]
This will be fixed by #225. Let's track conversation there.
